import_data_into_db.py

```diff
@@ -0,0 +1,58 @@
+import csv
+
+from sqlalchemy.orm import sessionmaker
+
+from models import Anime, engine
+
+Session = sessionmaker(bind=engine)
+
+session = Session()
+
+
+def populate_db(row):
+    """Funtion adds a row to sqlite DB
```

Comment:
If you use the mentioned way, I don't think this function is worth it. But you can keep it if you want.